internal/provider/cluster_resource.go

```diff
@@ -124,11 +124,27 @@ func (r *clusterResource) Schema(
 				},
 				Attributes: map[string]schema.Attribute{
 					"spend_limit": schema.Int64Attribute{
-						Required: true,
-						PlanModifiers: []planmodifier.Int64{
-							int64planmodifier.UseStateForUnknown(),
+						Optional:            true,
+						MarkdownDescription: "Spend limit in US cents.",
```

Comment:
Note that I'm not using plan modifiers. The reason is that _either_ spend limit or usage limits must be specified, but not both. `UseStateForUnknown` caused awkwardness when trying to set resource limits when a spend limit was already in place. The updated config would only specify resource limits, but Terraform would fill in the spend limit based on the previous state. This would then cause a server-side error, b/c both spend limit and resource limits were specified. Let me know if dropping `UseStateForUnknown` isn't the best way of avoiding this problem.